save-core/src/jvmTest/kotlin/org/cqfn/save/core/utils/ProcessBuilderInternalTest.kt

```diff
@@ -0,0 +1,102 @@
+package org.cqfn.save.core.utils
+
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+@Suppress("LOCAL_VARIABLE_EARLY_DECLARATION",
+    "SAY_NO_TO_VAR",
+    "MISSING_KDOC_TOP_LEVEL",
+    "MISSING_KDOC_CLASS_ELEMENTS",
+    "MISSING_KDOC_ON_FUNCTION"
+)
+class ProcessBuilderInternalTest {
+    private val processBuilder = ProcessBuilder()
+
+    @Test
+    fun `check stdout`() {
+        val actualResult = processBuilder.exec("echo something", null)
+        val expectedCode = 0
+        val expectedStdout = "something"
+        val expectedStderr: List<String> = emptyList()
+        assertEquals(expectedCode, actualResult.code)
+        // posix `system()` and JVM process builder returns lines with different whitespaces, so we cut them
+        assertEquals(expectedStdout, actualResult.stdout[0].trimEnd())
```

Comment:
Is it needed only in tests or should we also trim in the actual logic?

Reply:
It is corresponds to https://github.com/cqfn/save/pull/49#discussion_r622840750. Windows add extra white spaces and newlines. So yes, it's just for tests.

However, it's getting quite annoying. May be we can create some issue on this subject. I believe there is exist more
elegant workaround

Reply:
Yes, I agree, feel free to open an issue and describe what you have already found during your investigation